pkg/controllers/managedclusteraddons/managedclusterinfo.go

```diff
@@ -0,0 +1,209 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package managedclusteraddons
```

Comment:
https://github.com/stolostron/multicloud-operators-foundation/blob/5eddd28d21f70aa5e6b14e1385a78c53e519c228/cmd/controller/app/server.go#L179

just like Policy and managedserviceaccount addon. Do not need to rewrite. I did it before because there is package conflicts in OCM. since it is in stolostron, it should be OK. just have a try.

Reply:
I understand now, before I just replaced the clustermanagedaddon codes, 

and now have removed the rewriting codes of managedclusterinfo and imported the managedclusteraddon lib from the `stolostron/multicloud-operators-foundation`